src/util/headerValidation/.gitignore

```diff
@@ -0,0 +1,3 @@
+# The tests store failures in a file for easier inspection- this is because test failures for
+# fuzzing can be large
+testfail*/
```

Comment:
why is this .gitignore here?

Reply:
Because the header validation code is fuzz tested the output can be quite large, so failures are printed to a file also, to make it a bit easier to inspect. This gitignore file refers to those output files.